structure_tests/structure_test_v1.go

```diff
@@ -121,3 +98,89 @@ func (st StructureTestv1) RunFileContentTests(t *testing.T) {
 		}
 	}
 }
+
+// currently a hack to replace PWD in output/error until we figure
+// out a better way to handle this.
+func SubstituteEnvVars(t *testing.T, lists ...*[]string) {
+	env_var := "PWD"
+	value := os.Getenv(env_var)
+	if value == "" {
+		t.Fatalf("Variable %s not found in environment!", env_var)
+	}
+	for _, list := range lists {
+		for i := range *list {
+			str := (*list)[i]
+			(*list)[i] = strings.Replace(str, "$"+env_var, value, -1)
+		}
+	}
+}
+
+func ProcessCommand(t *testing.T, fullCommand []string, checkOutput bool) (string, string) {
+	var cmd *exec.Cmd
+	command := fullCommand[0]
+	flags := fullCommand[1:]
```

Comment:
Are you sure? It looks like you might bet a slice bounds out of range error:
https://play.golang.org/p/RNZXkxIIsi